<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTMengeClass</class>
 <widget class="QMainWindow" name="QTMengeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人群疏散仿真</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>选择仿真场景</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filePathInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="select_pbn">
        <property name="text">
         <string>选择场景</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QPushButton" name="strategic_pbn">
        <property name="text">
         <string>选择战略层模块</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>当前选择的战略层模块为：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>当前选择的战术层模块为：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="tactical_pbn">
        <property name="text">
         <string>选择战术层模块</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>当前选择的操作层模块为：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="operation_lineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="operation_pbn">
        <property name="text">
         <string>选择操作层模块</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>当前选择的环境层模块为：</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="environment_lineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="environment_selcet_pbn">
        <property name="text">
         <string>选择环境层模块</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="strategic_lineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>当前选择的可视化模块为：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="viewer_lineEdit"/>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="viewer_pbn">
        <property name="text">
         <string>选择可视化模块</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="tactical_lineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QPushButton" name="startMengeButton">
        <property name="text">
         <string>开始仿真</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>已选模块详情：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="info_text">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>655</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string/>
    </property>
    <addaction name="actionXml"/>
    <addaction name="actionTEST"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="GlobalPlanMaker"/>
    <addaction name="TacticPlanMaker"/>
    <addaction name="OperationPlanMaker"/>
    <addaction name="EnvironmentMaker"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionXml">
   <property name="text">
    <string>XML</string>
   </property>
  </action>
  <action name="actionTEST">
   <property name="text">
    <string>TEST</string>
   </property>
  </action>
  <action name="GlobalPlanMaker">
   <property name="text">
    <string>制作战略模块</string>
   </property>
  </action>
  <action name="TacticPlanMaker">
   <property name="text">
    <string>制作战术模块</string>
   </property>
  </action>
  <action name="OperationPlanMaker">
   <property name="text">
    <string>制作操作模块</string>
   </property>
  </action>
  <action name="EnvironmentMaker">
   <property name="text">
    <string>制作环境层模块</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtmenge.qrc"/>
 </resources>
 <connections/>
</ui>
